include/spirv-tools/optimizer.hpp

```diff
@@ -102,6 +102,12 @@ Optimizer::PassToken CreateNullPass();
 // Section 3.32.2 of the SPIR-V spec) of the SPIR-V module to be optimized.
 Optimizer::PassToken CreateStripDebugInfoPass();
 
+// Creates a eliminate-dead-functions pass.
+// A eliminate-dead-functions pass will remove all functions that are not entry
+// points, and cannot be reached from and entry point.  These functions are not
```

Comment:
"and" -> "an".
Also have to preserve functions reachable from a function with external linkage.